src/viur/core/modules/user.py

```diff
@@ -163,6 +161,12 @@ class UserSkel(skeleton.Skeleton):
         readOnly=True,
         defaultValue=0,
     )
+    # Authenticator OTP Apps (like Authy)
+    otp_secret_key = CredentialBone(
+        descr=u"OTP Secret key",
```

Comment:
avoid old-style unicode-notation
```suggestion
        descr="OTP Secret key",
```